scripts/rabbit_dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-list=( bionic  artful stretch buster trusty xenial )
```

Comment:
on arm systems I'd say "stretch is known not to work" (unless you're building lots of things yourself, which we can't support); on x86 I think I can only say "stretch is not known to work" (it may be possible, but given that moving to volttron 7 requires so much new installation anyway, anyone making that upgrade should take the opportunity to upgrade the distro as well, similar for new installs)... So I certainly agree that I'd like us to just not support stretch, I don't know if we have some other obligation that means we need to figure out a way to support it.